MovieApp/PresentationLayer/Favorites/FavoritesViewController+Design.swift

```diff
@@ -15,6 +15,9 @@ extension FavoritesViewController: ConstructViewsProtocol {
     func createViews() {
         navigationView = NavBarView()
         view.addSubview(navigationView)
+
+        moviePoster = MoviePoster()
```

Comment:
Why did you add this to this VC? For the screenshot or? Because it will probably be in some collection view so delete this.

Reply:
Just for the screenshot and testing, yes 😃